scripts/ipfs-reconnect-solo.sh

```diff
@@ -1,20 +1,29 @@
 #!/bin/bash
 
+# Detect whether Docker exists
+if command -v docker >/dev/null 2>&1; then
```

Comment:
@raymondkfcheung can we replace with argument check, e.g.: `./script/ipfs-reconnect-solo.sh docker`,
because at the remote server, I have docker and also local kubo, so just to check `command -v docker` is not enough

```suggestion
if <arg> == "docker" then
```

Reply:
> Updated to defaults to Docker

:+1: even better (at least for me :smile: )